test/common-algorithms/sorting/sequential/sequential_mergesort.chpl

```diff
@@ -0,0 +1,100 @@
+
+//merges two subarrays
+//First subarray is arr[l..m]
+//Second subarray is arr[m+1..r]
+
+proc merge(arr, left:int, mid:int, right:int){
```

Comment:
This is identical code to the parallel case, correct?  If so, you could merge the two and add a flag/threshold for when to use parallel or sequential (the threshold could also be used to show the heuristic you were describing, killing two birds with one stone)